KD_Lib/noisy/soft_random.py

```diff
@@ -154,8 +154,8 @@ def calculate_kd_loss(self, y_pred_student, y_pred_teacher, y_true):
         """
         loss = (1.0 - self.distil_weight) * F.cross_entropy(y_pred_student, y_true)
         loss += self.distil_weight * self.loss_fn(
-            F.log_softmax(y_pred_student / self.temp),
-            F.softmax(y_pred_teacher / self.temp),
+            F.log_softmax(y_pred_student / self.temp, dim=1),
+            F.softmax(y_pred_teacher / self.temp, dim=1),
```

Comment:
make  this `(self.distil_weight * self.temp * self.temp) * self.loss_fn(...)`. This generally gives faster convergence.

Reply:
Okie